web/src/inbox_ui.ts

```diff
@@ -1126,6 +1126,8 @@ class InboxTopicListWidget extends TopicListWidget {
                 }
 
                 channel_view_topic_widget.build();
+                // Also, update the left sidebar topics list for this channel.
+                topic_list.update_widget_for_stream(this.my_stream_id);
```

Comment:
Hmm, is this the only place we do a `stream_topic_history_util.get_server_history` that is missing some desired side effects?